activerecord/lib/active_record/inheritance.rb

```diff
@@ -37,6 +37,7 @@ module Inheritance
 
     included do
       # Determines whether to store the full constant name including namespace when using STI.
+      # This is true, by default.
```

Comment:
I noticed that in other files, the default value is documented. But, I can take it off.